src/Aspire.Dashboard/Components/Pages/Metrics.razor.cs

```diff
@@ -116,7 +116,7 @@ protected override void OnInitialized()
     protected override async Task OnParametersSetAsync()
     {
         var dashpages = await DashpagePersistence.GetDashpagesAsync(CancellationToken.None);
-        PageViewModel.Dashpages = dashpages;
+        PageViewModel.Dashpages = dashpages.Sort(DashpageDefinition.DisplayNameComparer);
```

Comment:
Can this just be:

```c#
dashpages.OrderBy(d => d?.DisplayName, StringComparer.Ordinal);
```

Also, as we show these items in two places in the UI, I would expect this ordering logic to appear twice.

Would be good to have a `StringComparisons.DashpageDisplayName` if we don't already.